### ISSUE TITLE
Support for viewing the file contents of units, services, timers etc.

### ISSUE THREAD

**Issue by kpbaks** [rgwood/systemctl-tui#26]
When I have a service which does not start properly it often helps me to look at its `.service` file to see how it is defined. A view or tab for this would be helpful.

**Comment by rgwood** [comment]
Yeah that's a good idea. Right now `systemctl-tui` can copy the unit file path to the clipboard to help with that, but it might be nice to be able to toggle between logs and the unit file.


![image](https://github.com/user-attachments/assets/16754d52-8e38-4da3-91af-21aef2a7fb1a)

**Comment by rgwood** [comment]
FYI in v0.3.9 I've added a new "Edit unit file" action that opens the unit file in `$EDITOR`. 

Still needs a bit of work to automatically reload the unit file (#29) but I think it should address this issue. Let me know if not.

**Comment by kpbaks** [comment]
Awesome, thank you!
